libmscore/fret.cpp

```diff
@@ -451,6 +465,11 @@ void FretDiagram::layout()
             return;
             }
       autoplaceSegmentElement();
+
+      // don't display harmony in palette
+      if (score() == gscore)
```

Comment:
FWIW, the more usual way we check for palette is "if (!!parent())".  Not saying it's better, but consistency has its merits.  I suppose we could factor that out as a function in the future, Element::isPalette() or some such.

Reply:
OK, thanks. There isn't enough space in the palette for harmonies, but I'll change the check to your suggestion.

Reply:
fixed by next commit